### PR TITLE
Lint Dockerfile files with Hadolint

### PR DESCRIPTION
**Pull request by phansys**

### CODE REVIEW

**Comment by greg0ire** [comment]
Since the plan is to release 12.0.0 as soon as #314 is merged (we need 3 approvals per PR for this repo), I think your new PRs should target 12.0.x so that we don't have to do another merge up.

**Review by ostrolucky — DISMISSED** [review]

**Comment by phansys** [comment]
>Since the plan is to release 12.0.0 as soon as https://github.com/doctrine/coding-standard/pull/314 is merged (we need 3 approvals per PR for this repo), I think your new PRs should target 12.0.x so that we don't have to do another merge up.

I've retargeted all my PRs to 12.0.x. Thank you!

File: .github/workflows/continuous-integration.yml
Comment (greg0ire) [inline .github/workflows/continuous-integration.yml]: This is going to run on every modification of every file. Maybe there are files that should not trigger this job?

Comment (phansys) [reply .github/workflows/continuous-integration.yml]: In this case, we should use a different workflow, as the `paths` config is not allowed in individual jobs.
Please, confirm.

Comment (greg0ire) [reply .github/workflows/continuous-integration.yml]: Indeed. Please go ahead and create a separate workflow.

File: .github/workflows/ci-dockerfile.yml
Comment (greg0ire) [inline .github/workflows/ci-dockerfile.yml:8]: The workflow itself should be in the list.

**Review by greg0ire — APPROVED** [review]

**Review by ostrolucky — APPROVED** [review]
Out of last 7 PRs that I was asked to review, only one was changing the actual coding standard rules. I don't see why we need so many approvals for CI plumbing PRs. There should be something done about this extra noise, otherwise I will disengage and leave @teams/coding-standard-approvers

**Comment by greg0ire** [comment]
:thinking: it is indeed weird that this kicked in: https://github.com/doctrine/coding-standard/blob/db39894b01cf5610a939387a1b79e2ae9dd85065/.github/CODEOWNERS#L1

**Review by SenseException — APPROVED** [review]

**Pull request merged**
